src/components/Layout/LayoutHero.tsx

```diff
@@ -0,0 +1,18 @@
+import { forwardRef, ReactElement } from 'react'
+import { Slot } from '@spark-ui/slot'
+import { cx } from 'class-variance-authority'
+
+export interface LayoutHeroProps {
+  asChild?: boolean
+  children: ReactElement
+  className?: string
+}
+export const LayoutHero = forwardRef<HTMLElement, LayoutHeroProps>(
+  ({ children, className }, forwardedRef) => (
+    <Slot ref={forwardedRef} className={cx('min-h-[50dvh] w-screen grow', className)}>
```

Comment:
IMO `hero` shouldn't be part of the layout every page can have its own hero element setup

Reply:
I think it should not. The hero section (like every section) in the  page layout must be structured following the same pattern. it willl reduce the anoying layout shifting when navigating between different urls